yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
+-- > followRedirect
+followRedirect :: Yesod site
+               =>  YesodExample site (Either T.Text ())
```

Comment:
yes, good call. I was trying to preserve the idea that Left was an error with some context, rather than inverting the typical semantics of Maybe, but given I have the slot I might as well pass back something useful.